releasenotes/notes/add-warning-ntp-offset-too-high-4ecabb07d0fd3417.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added a warning message on agent status command and status gui
+    tab when ntp offset is too large and may result in metrics
+    ignored by the main application.
```

Comment:
```suggestion
    ignored by the web application.
```

Reply:
Should I also mention "web application" in the warning message?